requirements/requirements.in

```diff
@@ -56,7 +56,7 @@ socketpool==0.5.3
 markdown==2.2.1
 sqlagg==0.16.2
 django-redis==4.11.0
-redis==3.2.1
+redis==3.3.0
```

Comment:
Might it not be better to upgrade to 3.3.11 if we want not to jump multiple major versions?

I also forget if there's a specific reason we can't jump multiple versions in this case, or if it's just a general precaution. The latest version is redis==3.5.3